src/resultscallercalleepage.cpp

```diff
@@ -234,21 +233,17 @@ void ResultsCallerCalleePage::onSourceMapContextMenu(QPoint point)
 
     QMenu contextMenu;
     auto* viewCallerCallee = contextMenu.addAction(tr("Open in Editor"));
-    auto* action = contextMenu.exec(QCursor::pos());
-    if (action == viewCallerCallee) {
-        emit navigateToCode(location.path, location.lineNumber, 0);
-    }
-}
+    connect(viewCallerCallee, &QAction::triggered, this,
+            [this, location] { emit navigateToCode(location.path, location.lineNumber, 0); });
+
+    auto line = index.data(SourceMapModel::FileLineRole).value<Data::FileLine>();
+    auto disassemblyAction = contextMenu.addAction(tr("Disassembly"));
+    connect(disassemblyAction, &QAction::triggered, this, [this, line] {
+        if (m_symbol.isValid())
+            emit jumpToSourceCode(m_symbol, line);
+    });
 
-void ResultsCallerCalleePage::onSourceMapActivated(const QModelIndex& index)
-{
-    const auto location = toSourceMapLocation(index);
-    if (location) {
-        emit navigateToCode(location.path, location.lineNumber, 0);
-    } else {
-        const auto fileLine = index.data(SourceMapModel::FileLineRole).value<Data::FileLine>();
```

Comment:
this old validation code and message got lost in translation now no?

Reply:
yes, fixed it